tests/cypress/support/commands_review_pipeline.js

```diff
@@ -157,7 +155,7 @@ Cypress.Commands.add('submitReview', (decision, user) => {
     cy.get('.cvat-submit-review-dialog').within(() => {
         cy.contains(new RegExp(`^${decision}$`, 'g')).click();
         if (decision === 'Review next') {
-            cy.intercept('GET', `/api/v1/users?is_active=true&search=${user}&limit=10`).as('searchUsers');
+            cy.intercept('GET', `/api/v1/users?search=${user}&limit=10&is_active=true`).as('searchUsers');
```

Comment:
@dvkruchinin 
Cypress tests failed because of this line. We do not really control the order of query parameters in these URLs (and probably we do not want to control it). Does cypress require exact order or maybe do you have ideas on how to improve it?

Reply:
Yes, the order must be followed. But in this case I think it can be processed through wildcard
```
cy.intercept('GET', '/api/v1/users*').as('searchUsers');
```